packages/react/src/indicator-dot/docs/overview.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Indicator dot
+description: Indicator dots are small decorative indicators used to call attention to an item, such as an unread message.
```

Comment:
Can we get "badge" in to this sentence somehow so this shows up in a search for "badge"

Reply:
```suggestion
description: Indicator dots are small decorative badge-like indicators used to call attention to an item, such as an unread message.
```